pages/c/_cluster/explorer/ResourceGauge.vue

```diff
@@ -0,0 +1,133 @@
+<script>
+import GraphCircle from '@/components/graph/Circle';
+import GradientBox from '@/components/GradientBox';
+
+export default {
+  components: { GradientBox, GraphCircle },
+  props:      {
+    name: {
+      type:     String,
+      required: true
+    },
+    total: {
+      type:     Number,
+      required: true
+    },
+    useful: {
+      type:     Number,
+      required: true
+    },
+    primaryColor: {
+      type:     String,
+      required: true
+    },
+    warningCount: {
+      type:    Number,
+      default: 0
+    },
+    errorCount: {
+      type:    Number,
+      default: 0
+    },
+    location: {
+      type:    Object,
+      default: null
+    }
+  },
+
+  computed: {
+    percentage() {
+      if (this.total === 0) {
+        return 0;
+      }
+
+      return this.useful / this.total;
+    },
+    clickable() {
+      return !!this.location;
+    },
+    secondaryColor() {
+      return this.$store.getters['prefs/theme'] === 'dark' ? 'rgba(74, 75, 82, 0.5)' : 'rgba(255, 255, 255, 0.15)';
```

Comment:
CSS variables.  Don't assume a particular theme, or that there's only two of them, or look at the name of it.

Reply:
You're totally right. Done.